### DISCUSSION TITLE
reactJSS

### DISCUSSION THREAD

**Discussion by farhanbaeee** [community/community#80301] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

why reactJs?

**Comment by fmpangestu** [comment]
> ### Select Topic Area
> General
> 
> ### Body
> why reactJs?

react is powerfull

**Comment by farhanbaeee** [comment]
me learning react step

**Reply by fmpangestu** [reply]
good job

**Reply by fmpangestu** [reply]
gooddd

**Comment by farhanbaeee** [comment]
JavaScript or Go
